src/bin/vhost_user_blk.rs

```diff
@@ -25,8 +25,8 @@ fn main() {
                 .long("block-backend")
                 .help(
                     "vhost-user-block backend parameters \"image=<image_path>,\
-                     sock=<socket_path>,readonly=true|false,\
-                     direct=true|false\"",
+                     sock=<socket_path>,num_queues=<number_of_queues>,\
+                     readonly=true|false,direct=true|false\"",
```

Comment:
@yangzhon I think you need to update the help for the "cloud-hypervisor" binary too. You only updated the deprecated binary.

Reply:
The below one?  
 ```
.arg(
            Arg::with_name("vhost-user-blk")
                .long("vhost-user-blk")
                .help(
                    "Vhost user Block parameters \"sock=<socket_path>,\
                     num_queues=<number_of_queues>,\
                     queue_size=<size_of_each_queue>, \
                     wce=<true|false, default true>\"",
                )
                .takes_value(true)
                .min_values(1)
                .group("vm-config"),
        )
```

The num_queues=<number_of_queues> has been added into cloud-hypervisor binary, although it is only set to 1 before.

Reply:
@yangzhon https://github.com/cloud-hypervisor/cloud-hypervisor/blob/7cb61d396032aa982e974c36d205aecdd43d67a9/src/main.rs#L286

The vhost_user_blk (and _net) binaries will be removed after 0.5 and the backends run via the cloud-hypervisor binary.

Reply:
@rbradford ,

Thanks a lot! I will add a patch for this block-backend. 
By the way, the Google_Summer_of_Code_2019 of Qemu is also implementing one vhost user block backend in the qemu.